djorg/settings.py

```diff
@@ -0,0 +1,84 @@
+"""
+Django settings for djorg project.
+Generated by 'django-admin startproject' using Django 2.1.4.
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+import os
+from decouple import config
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+SECRET_KEY = config('SECRET_KEY')
+DEBUG = config('DEBUG', cast=bool)
+ALLOWED_HOSTS = []
+INSTALLED_APPS = [
+    'notes.apps.NotesConfig',

```

Comment:
I'm assuming this works, but I'm curious as to why you have this here instead of 'notes'.